src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -40,7 +40,7 @@ public function __construct(LoaderResolverInterface $resolver, array $defaultOpt
     /**
      * {@inheritdoc}
      */
-    public function load($resource, $type = null)
+    public function load($resource, string $type = null)
```

Comment:
This will break compatibility with Config 4.4. Please either revert this change or update FrameworkBundle‘s composer.json accordingly.

Reply:
Good catch. Fixed.